doc/sphinx-guides/source/api/native-api.rst

```diff
@@ -5145,6 +5144,10 @@ List a saved search by database id. ::
 
   GET http://$SERVER/api/admin/savedsearches/$id
 
+Delete a saved search by database id. The ``unlink=true`` query parameter unlink links (Linked dataset or Dataverse collection) related to the deleted saved search. This parameter should be well considered as you cannot know if the saved search created the links or if someone else did via other API. Also, it may be followed ``/makelinks/all`` depending on the need if other saved searches could recreate some deleted links or by reindexing some Dataverse or Dataset. ::
```

Comment:
I'm having trouble parsing the sentence that starts with "Also, it may be followed".

@luddaniel can you please clarify? Thanks.

Reply:
Thanks, I just added a new suggestion. I hope I'm getting closer! 😅 Please take a look.